src/dtls.c

```diff
@@ -250,23 +291,30 @@ static int TlsTicketIsValid(WOLFSSL* ssl, WolfSSL_ConstVector exts,
     if (tlsxSessionTicket.size > SESSION_TICKET_LEN)
         return 0;
     XMEMCPY(tempTicket, tlsxSessionTicket.elements, tlsxSessionTicket.size);
-    ret = DoDecryptTicket(ssl, tempTicket, (word32)tlsxSessionTicket.size, &it);
+    ret = DoDecryptTicket((WOLFSSL*)ssl, tempTicket,
+                          (word32)tlsxSessionTicket.size, &it);
     if (ret != WOLFSSL_TICKET_RET_OK && ret != WOLFSSL_TICKET_RET_CREATE)
         return 0;
+    /* Store info for later */
+#if defined(WOLFSSL_TLS13) && defined(HAVE_SESSION_TICKET)
```

Comment:
we shouldn't parse the TLSX_SESSION_TICKET in DTLS 1.3 because the extension isn't allowed.

Reply:
Good catch. I also changed it so that the resumption info is checked for DTLS 1.3 since it affects cipher negotiation.